src/kinetic.ml

```diff
@@ -231,10 +245,10 @@ let network_receive ic trace =
   Lwt_io.read_into_exactly ic proto_raw 0 proto_ln >>= fun () ->
   begin
     if trace
-    then Lwt_log.debug_f
+    then Lwt_log.info_f
```

Comment:
this will result in plenty of info level logging for alba too?

Reply:
only if tracing is on, and that can be decided by alba.